purefa/dashboards/purefa_overview.json

```diff
@@ -0,0 +1 @@
eseries","queries":[{"query":"avg:purefa.host.performance_iops{dimension:read,$ArrayName,$Env} by {host,fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":2,"y":14,"width":5,"height":3}},{"id":7308391345126890,"definition":{"title":"Average Top 5 Hosts by Array - Write IOPS","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Write IOPS","formula":"top(query1, 5, 'mean', 'desc')"}],"response_format":"timeseries","queries":[{"query":"avg:purefa.host.performance_iops{dimension:write,$ArrayName,$Env} by {host,fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":7,"y":14,"width":5,"height":3}},{"id":3816947770113226,"definition":{"title":"Average Top 5 Hosts by Array - Mirrored Writes IOPS","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Mirrored Writes IOPS","formula":"top(query1, 5, 'mean', 'desc')"}],"response_format":"timeseries","queries":[{"query":"avg:purefa.host.performance_iops{dimension:mirrored_write,$ArrayName,$Env} by {host,fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":2,"y":17,"width":5,"height":3}},{"id":8966512506830154,"definition":{"title":"Mirrored Write Bandwidth by Array","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value"],"type":"timeseries","requests":[{"formulas":[{"alias":"Mirrored Write Bandwidth","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:purefa.array.performance_bandwidth_bytes{dimension:mirrored_write,$ArrayName,$Env} by {fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":7,"y":17,"width":5,"height":3}}],"template_variables":[{"name":"Env","default":"*","prefix":"env","available_values":[]},{"name":"ArrayName","default":"*","prefix":"fa_array_name","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"a9w-uji-pru"}
```

Comment:
Thanks for the dashboard updates! It's looking good. Just a few nits:
- The text widgets on the left are getting cut off, I think that expanding them a bit further to the left would help
- The logo isn't loading for some reason when I paste the dashboard JSON. Is this happening on your end? It may be having trouble loading the image from GitHub.

Reply:
This is becuase the PR hasn't yet merged. The URL is pointing to where the logo will be when the merge completes.

Reply:
Got it, thanks. The dashboard should be good to go then.